### ISSUE TITLE
Score for background and foreground 

### ISSUE THREAD

**Issue by Turlan** [kkanshul/finegan#4]
Hi, thanks for releasing code. It's a great job. But I have some tiny questions about the classification score of background. In line 337 of model.py, the background score is set to 0
`classi_score = self.uncond_logits1(x_code) # Background vs Foreground classification score (0 - background and 1 - foreground)`
It's consistent with your statement in your paper. But in lines 227 and 330 of trainer.py, the score of background seems to be set to 1.
`errD_real_uncond_classi = criterion(real_logits[0], weights_real)  # Background/foreground classification loss`
`errG_classi = criterion_one(outputs[0], real_labels) # Background/Foreground classification loss for the fake background image (on patch level)`
So, I am confused, could you point anything wrong about my understanding, that will be very helpful, thank you.

**Comment by utkarshojha** [comment]
Hi, thanks for pointing that out. 
Basically our code associates background class with 1 and foreground with 0. But while explaining (ex. comment in line 337 of model.py and in the paper) we've associated 0 and 1 with the opposite classes. We'll correct the comment and the text in the paper, but it won't matter in the way we train FineGAN, because ultimately we want to learn a binary classifier which learns to associate one of the classes (either background or foreground) to 0 and the other to 1.

Thanks~

**Comment by Turlan** [comment]
Thanks for reply, it's reasonable now.
